bettermdptools/envs/binning.py

```diff
@@ -0,0 +1,96 @@
+"""
+Generated using ChatGPT.
+"""
+
+def generate_bin_edges(range_limit, n_bins, width_ratio, center=True):
+    """
+    Generates bin edges for a symmetric range with exponentially increasing/decreasing bin widths.
```

Comment:
It might be good to use the same style of docstrings as the rest of the code (example): https://github.com/jlm429/bettermdptools/blob/master/bettermdptools/algorithms/planner.py

Reply:
Thanks!

Reply:
Thanks!